src/main.cpp

```diff
@@ -53,6 +54,13 @@ static void usage(std::ostream& out) {
     out << "  -v              enable verbose output\n";
 }
 
+// write(2) is safe to call inside signal handler.
+#define write1(x) write(1, x, sizeof(x) - 1)
+static void on_sighup(int _unused)
+{
+	(void)write1("Laminar don't have config to reload, SIGHUP is ignored\n");
```

Comment:
- I think stderr is more appropriate for this message than stdout
- For readability use `STDOUT_FILENO`,`STDERR_FILENO` instead of 1,2
- Message should not assume that `SIGHUP` is for reloading config. While common, it is not standard. Suggest simply `"Caught SIGHUP, ignoring.\n"`
- Global macro can be avoided with a function-local variable, should compile to the same:
```
constexpr const char msg[] = "...";
write(STDERR_FILENO, msg, sizeof(msg) - 1);
```